karapace/protobuf/kotlin_wrapper.py

```diff
@@ -10,18 +10,18 @@ def trim_margin(s: str) -> str:
     lines = s.split("\n")
     new_lines = list()
 
+    if not lines[0].strip():
```

Comment:
Btw, it seems this is being used to trim left space, there are a few alternatives for this which would remove the need to maintain and remove bugs from an util:

- Use [dedent](https://docs.python.org/3.6/library/textwrap.html#textwrap.dedent):

```python
>>> import textwrap
>>> s = """\
...    class A:
...        a: int\
... """
>>> print(textwrap.dedent(s))
class A:
    a: int
````
- Leave the whitespace in the left, the parser should ignore it
- Define it at the top-level, so the extra whitespace won't result in clutter